src/main/java/com/specialwarriors/conal/user/controller/UserController.java

```diff
@@ -32,7 +32,7 @@ public String index() {
     @GetMapping("/home")
     public String home(Model model, @SessionAttribute("userId") Long userId) {
 
-        User user = userService.getUserByUserId(userId);
+        User user = userService.getUser(userId);
```

Comment:
```suggestion
        User user = userService.findById(userId);
```
service의 메서드명을 repository에 맞추자는 예시는 위같은 형식을 의미했습니다. `getUser`는 어떤 파라미터를 바탕으로 조회하는 지 의미를 표현함에 있어 메서드명이 모호하다고 생각합니다.